sympy/polys/matrices/domainscalar.py

```diff
@@ -104,6 +104,9 @@ def __pow__(self, n):
     def __pos__(self):
         return self.new(+self.element, self.domain)
 
+    def __neg__(self):
+        return self.new(-self.element, self.domain)
```

Comment:
I'm considering removing the `DomainScalar` class. It would only be really useful if it was used throughout the Domain system. Since it isn't it is mostly just an extra complication. The man benefit is that it allows unification of scalar domains with the ground domain of a matrix. Maybe there is a better way to handle that though.